src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -337,6 +341,28 @@
   &.pf-m-danger {
     --#{$menu-toggle}--BorderColor: var(--#{$menu-toggle}--m-danger--BorderColor);
     --#{$menu-toggle}__status-icon--Color: var(--#{$menu-toggle}--m-danger__status-icon--Color);
+
+    @media screen and (prefers-reduced-motion: no-preference) {
+      transform: translate3d(0, 0, 0);
+      animation-name: menu-toggle-m-danger-motion;
+      animation-duration: var(--#{$menu-toggle}--m-danger--AnimationDuration--Transform);
+      animation-timing-function: var(--#{$menu-toggle}--m-danger--AnimationTimingFunction--Transform);
+      animation-fill-mode: both;
+      
+      .#{$menu-toggle}__status-icon {
+        color: var(--#{$menu-toggle}__status-icon--Color, inherit);
+        opacity: 1;
+        transition: 
+          opacity 
+            var(--#{$menu-toggle}__status-icon--m-danger--TransitionDuration--Opacity)
+            var(--#{$menu-toggle}__status-icon--m-danger--TransitionDuration--Opacity);
+        // stylelint-disable max-nesting-depth
+        @starting-style {
+          opacity: 0;
+        }
```

Comment:
Switch to using keyframes because @starting-style isn't fully supported by Firefox